prep-html-skeleton/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html> </html>
```

Comment:
The closing tag for the `html` element should be the last thing in your document. The `html` element is meant to hold all content for a HTML document.